manifests/params.pp

```diff
@@ -106,6 +106,7 @@
         'php5-ldap',
         'php5-mysql',
         'php5-pgsql',
+        'mysql-client-core-5.5',
```

Comment:
yeah, not fond using a module to provide only a client. I use the mysql module to manage my db servers.